### DISCUSSION TITLE
customer name over 2 (or more) rows

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3547] [General]
Sometimes, our customers have very long names like "something fancy Ltd., Zweigniederlassung Deutschland" 
When we create invoices for such a customer, the name will be printed in one row and thus, not be seen in the window of the envelope anymore.
![image](https://user-images.githubusercontent.com/51003649/191061200-c248c30d-6cb0-4b07-9f6d-141c57ac9127.png)

Would it be possible to have a line break (like in the address field or description field?)

![image](https://user-images.githubusercontent.com/51003649/191058929-6c3fc828-ea41-4af9-8230-3329958bc0a0.png)

**Comment by kevinpapst** [comment]
That is a perfect example for a "meta/custom field".
Create one for customers with a text box and use its content for the invoice. 
If the content is empty, use the nomal custom name.

**Comment by BeckeBauer** [comment]
I will give it a try. But I assume that I need to also adjust my invoice templates. I use word templates (docx) - is there an explanation how to include the check in the docx if the meta field is filled and if not to use the customer name instead?

**Reply by kevinpapst** [reply]
I don't think DOCX supports that. Just fill the field for all (new) customers 😁

**Reply by BeckeBauer** [reply]
I think that I will stick (and stuck) with current system... To much hazle to change all customers for one occasional line break  in the address field...

**Reply by kevinpapst** [reply]
Can't you tell your DOCX template to wrap at a certain position? 
Sound like a task that a Word processing software should be able to handle. 
That issue happens with Serienbriefen all the time...

**Reply by BeckeBauer** [reply]
wow - I am really stupid; that is of course the solution (the address field in word can be manipulated in any way I want...)
